manifests/01_cluster_csi_driver_crd.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why do we need this CRD here if it's in openshift/api?

I also have it in the EBS operator. It would be nice to have it in a common place.

Would linking this file to ./vendor/github.com/openshift/api/... reasonable?

Reply:
Something still needs to apply the CRD and it's CVO when it sees it in CSO's `manifests/`. You see there is `01_crd_operator.yaml` with the `Storage` CRD too.

About linking, I am not so sure, copy has always worked. Maybe as a separate PR?

Reply:
Added TODO comment